packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/RequestLoggingRequestFilter.java

```diff
@@ -0,0 +1,22 @@
+package software.uncharted.terarium.hmiserver;
+
+import lombok.extern.slf4j.Slf4j;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
```

Comment:
Are those librairies what allows you to add your function onto a Request?

Reply:
They're interfaces provided by JAX-RS to do things like add headers, implement authentication, etc.